packages/flutter/lib/src/material/drawer.dart

```diff
@@ -492,7 +522,13 @@ class DrawerControllerState extends State<DrawerController> with SingleTickerPro
   final GlobalKey _gestureDetectorKey = GlobalKey();
 
   ColorTween _buildScrimColorTween() {
-    return ColorTween(begin: Colors.transparent, end: widget.scrimColor ?? Colors.black54);
+    final DrawerThemeData drawerTheme = DrawerTheme.of(context);
+    return ColorTween(
+      begin: Colors.transparent,
+      end: widget.scrimColor
+          ?? drawerTheme.scrimColor
```

Comment:
This line could just be `DrawerTheme.of(context).scrimColor` (we can leave out the variable).

Reply:
Done